### PR TITLE
JSPM Lookups

### PR DESCRIPTION
**Pull request by swernerx**
Added support for JSPM based lookups.

I have not yet any tests for jspm lookup in this package, but in the origin/dependency project (would require some jspm installs which I am not sure should be done here).

### CODE REVIEW

File: lib/resolve-id.js [hunk 1]
```diff
@@ -23,7 +24,7 @@ module.exports = function(id, base, options) {
     basedir: base,
     moduleDirectory: moduleDirectories,
     paths: paths,
-    extensions: [ ".css" ],
+    extensions: [ ".css", ".sss" ],
```
Comment (MoOx) [inline lib/resolve-id.js]: Please don't add this. You can handle this via option.

File: package.json [hunk 2]
```diff
@@ -18,7 +18,9 @@
     "lib"
   ],
   "dependencies": {
+    "jspm": "^0.17.0-beta.13",
```
Comment (MoOx) [inline package.json]: Maybe devdeps?

File: .eslintignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+jspm.browser.js
```
Comment (MoOx) [inline .eslintignore]: why don't you lint those files?

File: lib/resolve-id.js (hunk 1 above)
Comment (swernerx) [inline lib/resolve-id.js]: Okay, I'll change it back.

File: package.json (hunk 2 above)
Comment (swernerx) [inline package.json]: That seems right. I'll change it.

File: .eslintignore (hunk 3 above)
Comment (swernerx) [inline .eslintignore]: Mostly because they are generated/modified when using jspm. They are not really human-controlled.

Comment (MoOx) [inline .eslintignore]: so sshould not be add in git?

**Comment by swernerx** [comment]
I reduced the integration for jspm a bit. This was mainly a try to solve the test puzzle for jspm. The current versions seems to be the minimum viable one to add jspm packages resolving...

File: package.json [hunk 4]
```diff
@@ -28,9 +29,20 @@
...
   },
+  "jspm": {
+    "name": "postcss-import",
+    "main": "index.js",
+    "browser": {
+      "./lib/load-content": "@empty",
+      "./lib/resolve-id": "@empty"
+    },
+    "dependencies": {},
+    "devDependencies": {}
```
Comment (MoOx) [inline package.json]: are the empty field required?

Comment (swernerx) [inline package.json]: You are really caring about the details, are you? - That's welcome.

Comment (swernerx) [inline package.json]: I will try with leaving them out...

**Comment by swernerx** [comment]
Looks good without dependencies and devDependencies.

**Comment by swernerx** [comment]
Would it be possible to merge this into master and do a new release? Or is there anything outstanding we have to work before this could happen?

**Comment by MoOx** [comment]
Released as 8.1.1

**Comment by jedrichards** [comment]
Adding `pkg-resolve` into the dependency tree caused my builds to fail with a ton of security warnings when I just upgraded to the latest `postcss-import`. I know these security warnings aren't necessarily relevant for packages used in build processes as opposed to outward facing webapps, but still thought you might be interested.

screen shot 2016-05-18 at 16 45 31

Also it seems a slight shame that adding jspm support means that all users have the additional complexity overhead of the jspm modules added to their dependency trees, even if they don't use jspm ... hence problems like the above ...

**Comment by MoOx** [comment]
I think we should move this to peerDependencies and only import in a jspm context. Will open an issue about that.

**Comment by MoOx** [comment]
https://github.com/postcss/postcss-import/issues/212

**Comment by MoOx** [comment]
@swernerx any thoughts about the security issues?

**Comment by aunz** [comment]
I use postcss-import in webpack postcss-loader
Addition of JSPM in version 8.1.1 or higher creates this error: `Module build failed: Error: Unable to fetch "C:\XXX\node_modules\babel-loader\index.js". Only file URLs of the form  allowed running in Node.`
A similar [issue](https://github.com/systemjs/systemjs/issues/306) was described in systemjs
Version 8.1.0 and below without JSPM has no such error
I think this happens only in Windows
My system is Windows 10 64 bit with node 6.2.0
Took me a while to figure this out, hope this can be useful for windows users who experience the same issue?

**Pull request merged**